Display Data to card

cmskorcz opened the pull request:
## Additions
- Added functions to dynamically display cards within `<div id='card-container'>`

## Changes
- Commented out the bottom portion of the 'to be completed' function to resolve flags
- Removed `$tokenButton` and `$tokenButton.on` as this will be removed once html skeleton is merged.

cmskorcz committed:
images of animal post to page if one is available

cmskorcz committed:
created getAnimalImage function

cmskorcz committed:
linked stylesheet and added some styling, cards now include name age breed and gender

cmskorcz committed:
animals display in cards with content inside

cmskorcz committed:
Merge branch 'develop' of github.com:ProgrammingAStorm/FurEverHome into feature/display-data

cmskorcz committed:
added displayAnimals func to success of getAnimals func

cmskorcz committed:
added tags to card with max of 3

cmskorcz committed:
added comments

cmskorcz committed:
resolved merge conflict from pull

ProgrammingAStorm reviewed (changes requested):
I tweaked the card design a little so if you could make sure its still consistent. It's in feature/favorites-integration, as at the time of writing this, it hasn't been push to develop. Also, make sure to give each card a "data-id" attribute equal to the pet's ID; my favorites functions are going to need that.
At line 290 there's a random getAnimals() call that I'm guessing needs to be assigned to a listener. Other than that it looks great. 👌

Also if you're feeling up to it, see if any of the custom styling can be done in Bulma. It's not a big deal and can be done later.
Thanks.

cmskorcz committed:
added id to cards fixed bug with timeout for getanimals

cmskorcz committed:
made changes for updated index

cmskorcz commented:
## Changes Made
- Added `data-id` attribute to cards with the id of animal
- Updated display functions to use indicated classes in the index file
- Instead of adding `mt-1` to the first tag, I added a `pb-1` to the div containing the tag `ul`. This was due to the first tag being displayed lower than the remaining tags when displayed side by side.
